src/main/java/uk/gov/hmcts/reform/blobrouter/data/eventsv2/EnvelopeEvent.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.blobrouter.data.eventsv2;
```

Comment:
personally not very strong on such package

Reply:
I'm open to  alternative names

Reply:
events can be enough? especially when former models etc will be removed anyway - there's no need to version now then. makes sense?

Reply:
there's already a package called 'events'

Reply:
oh you mean to put it in the existing one